### PR TITLE
fix: generate error docs in solo and upload as release artifact

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
## Description

This PR updates the `zxc-docs-automation.yaml` workflow to generate `.md` files for all `SoloErrors` and bundle them together with `solo-cli.md` in a single archive, which is uploaded as a release artifact during the release deployment process. This artifact will be consumed by a workflow in `solo-docs`.

### Related Issues

* Closes #
* Related to #

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[ ] This PR updated documentation
* \[ ] This PR added no TODOs or commented out code
* \[ ] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `package.json` changes have been explained to and approved by a repository manager
* \[ ] All related issues have been linked to this PR
* \[ ] All changes in this PR are included in the description
* \[ ] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[ ] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* TBD

The following was not tested:

* TBD

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.  NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                        |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

**Review by jan-milenkov — COMMENTED** [review]
Lgtm, @jeromy-cannon @JeffreyDallas please review

File: .github/workflows/zxc-docs-automation.yaml [hunk 1]
```diff
@@ -152,7 +152,7 @@ jobs:
         id: solo-cli-help
         run: |
           cd docs/site
-          SOLO_CI=true task build:cli:help
+          SOLO_CI=true task build:docs:content
```
Comment (jeromy-cannon) [inline .github/workflows/zxc-docs-automation.yaml:155]: see my comment in the solo-docs PR, would be good to have verified prior to merging and tagging to reduce multiple iterations.

I think Jan or I can attached the release artifact if you don't have access.

**Review by jeromy-cannon — APPROVED** [review]
